mittab/apps/tab/middleware.py

```diff
@@ -4,17 +4,22 @@
 from django.http import HttpResponseRedirect
 from django.core.cache import cache
 
-login_white_list = ('/accounts/login/', '/static/css/stylesheet.css',
+string_white_list = ('/accounts/login/', '/static/css/stylesheet.css',
```

Comment:
IMO we should just refactor this so that all strings are just regexes and get rid of the duplicate logic.

For example, `re.compile("^/accounts/login/$")`

Reply:
Refactored in the latest commit.